mock/mockmr/views.go

```diff
@@ -84,7 +86,10 @@ func (rc resultContainer) Len() int {
 }
 
 func (rc resultContainer) Less(i, j int) bool {
-	return rc.results[i].Key < rc.results[j].Key
+	keyI, _ := NewKeysFilter([]byte(rc.results[i].Key))
```

Comment:
We appear to ignore the value from this everywhere, we should probably do *something* with it. Especially as the filter will be nil on error.